src/app/dashboard/whitelist/components/editInvestorsForm.js

```diff
@@ -0,0 +1,42 @@
+// @flow
+
+import React, { Component } from 'react'
+import { Field, reduxForm } from 'redux-form'
+
+import { Form } from 'carbon-components-react'
+import { TextInput } from 'polymath-ui'
+import { required } from 'polymath-ui/dist/validate'
+
+export const formName = 'edit_investor_form'
+
+type Props = {}
+
+class EditInvestorsForm extends Component<Props> {
+
+  render () {
+    return (
+      <Form>
+        <Field
+          name='sell'
+          component={TextInput}
```

Comment:
You should date input from carbon components instead of text input. Same for the fields below

Reply:
fixed 

made a new component that does exactly this. it is in this repo right now, I will have to submit a PR to get it merged into polymath UI, and then from there I will reference polymath UI, and remove it from here .